qcodes/tests/drivers/test_ami430_pyvisasim.py

```diff
@@ -0,0 +1,413 @@
+import io
+import numpy as np
+import pytest
+from hypothesis import given, settings
+from hypothesis.strategies import floats
+from hypothesis.strategies import tuples
+import logging
+from typing import List, Dict
+
+import qcodes.instrument.sims as sims
+from qcodes.instrument_drivers.american_magnetics.AMI430 import AMI430_3D
+from qcodes.instrument_drivers.american_magnetics.AMI430_VISA import AMI430_VISA
```

Comment:
I think we should make the AMI430_VISA class in the this testing module. The folder /qcodes/instrument_drivers/american_magnetics/ should be reserved for actual drivers only.

Reply:
Fully agree. I'll move it.